src/components/Stars/Rating.tsx

```diff
@@ -0,0 +1,40 @@
+'use client';
+
+import { useState } from 'react';
+import Star from './Star';
+
+interface RatingProps {
+  initialRating: number;
+  onRatingChange?: (rating: number) => void;
+  isEditable?: boolean;
+}
+
+export default function Rating({ initialRating, onRatingChange, isEditable }: RatingProps) {
+  const STARS = [1, 2, 3, 4, 5];
+  const [rating, setRating] = useState<number>(Math.floor(initialRating));
+  const [hoverRating, setHoverRating] = useState<number>(0);
+
+  const handleRating = (rate: number) => {
+    if (isEditable && onRatingChange) {
```

Comment:
컴포넌트를 불러올 때 isEditable이 들어가 있으므로, isEditable은 조건에서 빼도 될 것 같습니다.
```suggestion
    if (onRatingChange) {
```

Reply:
네엡 다시보니 빼도 될 것 같아요 !!